.github/actions/auto-assign/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "private": true,
+  "name": "auto-assign",
+  "version": "0",
+  "exports": "./index.js",
+  "files": [
+    "src"
+  ],
+  "scripts": {
+    "pack": "ncc -m -o . build lib/index.js",
+    "build": "tsc && yarn run pack"
+  },
+  "devDependencies": {
+    "@octokit/webhooks-types": "6.9.0",
+    "@types/node": "^18.11.0",
+    "@vercel/ncc": "0.34.0",
+    "typescript": "^4.9.4"
```

Comment:
There is a newer version of typescript, what do you think about using the latest for this new package?

Reply:
haha whoops this is the wrong package.json, still valid technically but you can ignore.

Reply:
It's valid. Upgraded. Thanks for catching that.